discord-scripts/thread-management/auto-join.ts

```diff
@@ -23,7 +23,11 @@ import {
 // it to mention the right role. Discord's behavior in this scenario is not to
 // ping the role, but to add all its members to the thread.
 
-const CUSTOM_CHANNEL_ROLE = [{ channelName: "hiring", roleName: "PeopleOps" }]
+const CUSTOM_CHANNEL_ROLE: Record<string, string> = {
```

Comment:
Fwiw the one downside of `Record<string, string>` is that it removes our ability to name either string. Whereas we could do:

```typescript
const CUSTOM_ROLE_TAG_BY_CHANNEL: { [channelName: string]: string } = {
  ...
}
```

Mostly academic, but useful to know :) `channelName` here does nothing other than provide a hint to the programmer as to what the key of the object is meant to be.